apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/Program.cs

```diff
@@ -221,8 +222,10 @@ private class CustomStreamMethodInfo
 
             // Create a CompilationUnitSyntax from the Usings node of the original file, which will also contain the
             // copyright notice and generated code warnings in its leading trivia.
-            var compilationUnit =
-                CompilationUnit().WithUsings(syntaxTree.GetCompilationUnitRoot().Usings);
+            // We also need a using directive for GAX, so that we can use ResultWithUnwrappedExceptions.
+            var usings = syntaxTree.GetCompilationUnitRoot().Usings
+                .Add(UsingDirective(IdentifierName(typeof(gax::TaskExtensions).Namespace)));
```

Comment:
IdentifierName => ParseName

Reply:
That still should use ParseName as it is a QualifiedNameSyntax (uses '.' to qualify as opposed to '::' for AliasQualifiedNameSyntax). IdentifierName is just a simple identifier that could be used as a local name, for example.